docs/site-generator/Makefile

```diff
@@ -0,0 +1,155 @@
+# Default shell
```

Comment:
I'm OK with it, please don't forget about the monster you are hiding under the bed.